### PR TITLE
fix go.mod

### PR DESCRIPTION
**Pull request by patrick-ogrady**

### CODE REVIEW

File: go/avalanche.go [hunk 1]
```diff
@@ -19,9 +19,10 @@ package ledger_avalanche_go
 import (
 	"errors"
 	"fmt"
+
 	"github.com/btcsuite/btcd/btcec/v2"
 	"github.com/btcsuite/btcd/btcec/v2/ecdsa"
-	"github.com/zondax/ledger-go"
+	ledger_go "github.com/zondax/ledger-go"
```
Comment (StephenButtolph) [inline go/avalanche.go:25]: Minor nit - generally underscores a kinda frowned upon for import aliases... ex: https://github.com/uber-go/guide/blob/master/style.md#package-names

**Review by ftheirs — APPROVED** [review]
We can skip the check_version step from CI because the app code doesn't change with these modifications

**Comment by becominginsane** [comment]
LGTM.
@patrick-ogrady let me know if we should merge this as is.

@ftheirs agreed, but we can also change the base to `dev` and make the change part of the next merge

**Comment by patrick-ogrady** [comment]
We plan to ship this in the next AvalancheGo release (which may be in the next day or so). It would be great if you could merge it right to master so we don't have to rely on a commit from a PR.

**Pull request merged**